src/shared/components/Workspace/Queries/Query/query-component.less

```diff
@@ -0,0 +1,55 @@
+@import '../../../../lib.less';
+
+.query-component {
+  .cardlike();
+  background-color: #f5f6f7;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+  & > .infinite-scroll {
+    overflow: hidden;
+    overflow-y: auto;
+  }
+  & > .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: @text-color; // instead of default heading color
+  }
+  & > .header > .count {
+    font-size: 0.8rem;
+    font-weight: lighter;
+    flex-grow: 1;
+    margin-left: 1em;
+  }
+  & > .controls {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1em;
+    flex-wrap: wrap;
+  }
+  & > .controls > .search {
+    width: 60%;
+    margin-right: 2px;
+  }
+  & > .controls > .filter-dropdown {
+    width: calc(50% - 2px);
+    margin: 0.5em 0;
+  }
+  & > .infinite-scroll > .list .list-item {
+    margin: 0.5em 0;
+  }
+  &.-add {
+    cursor: pointer;
+    padding: 1em;
+    color: hsla(0, 9%, 91%, 0.8);
```

Comment:
should be a variable. Also, why hsla() here instead of rgba() which is closer to the hex notation used elsewhere?